transform-gcp-cost-tables.py

```diff
@@ -0,0 +1,82 @@
+"""
+Script that ingests a CSV cost table generated via the GCP UI and performs a
+transformation.
+
+We assume the input CSV file has the following columns (and are subject to changes
+by GCP):
+- Project name
+- Month
+- Subtotal ($)
+
+We aim to have an output CSV file with columns:
+- Project name
+- start-month
+- ...
+- end-month
+- Total
+
+where:
+- start-month...end-month are unique values from the Month column in the input file
+- Project name are unique entries from the input file
+- The Total column is the sum across all month columns for each project
+"""
+
+from pathlib import Path
+from sys import argv
+
+import pandas as pd
+
+# Get input file path from command line args
+input_filepath = Path(argv[1])
+
+# Read input file
+input_cost_table = pd.read_csv(
+    input_filepath,
+    usecols=["Month", "Project name", "Subtotal ($)"],
+)
+
+# Drop NaN values
+input_cost_table = input_cost_table.dropna()
+
+# Rename the projects column. This makes using the 'query' function later easier
```

Comment:
Filipe suggests:

> You can also rename a column with the .rename instead of copying and dropping. Not that different but it will look more "panda-ish."